docs/extensions/model-merger.md

```diff
@@ -0,0 +1,66 @@
+# Model Merger
+
+The model merge is used to automatically merge two models. Merge is not meant in the same way as in terms VCS systems like git. It‘s main use case is incrementalisation of importer at the moment but is not limited to it.
+## Model Merge Specification 
+
+The model merge needs a customizable specification of how to merge instances of a given concept. The DSL should support the user to write correct specifications for a set of concepts and allow to reuse specifications for instance when a concept is extended or interfaces are implemented. Putting this into a DSL rather than writing this in plain java allows us to evolve the runtime and the language independently. Inspirations can be taken from the new VCD aspect in MPS. While a much simpler use case it might still be useful input.
+
+### Concrete Syntax Example
+```
+Spec
+----
+<Concept> -> Id : (node) = node.name
+                         ---- Properties ----
+                         name -> left | right | manual (left, right, node) -> <function returning the new property value>
+                        prop 2 -> 
+                        ---- Children ----
+                        childA [0..1] -> left | right | drop | auto | manual (left, right, node) -> <function returning the node used for the child>
```

Comment:
I think we should write down somewhere what "left | right | drop | auto " means, so:
left: changes from left dominate over changes from right (?)
right: changes from right dominate over changes from left (?)
drop: as soon as a node changes, either from left or right, it is removed (?)
auto: (?)

Reply:
I agree. I will add a paragraph about the semantics.